### ISSUE TITLE
Strange sound

### ISSUE THREAD

**Issue by luizj** [phoboslab/jsmpeg#204]
Does anyone hear strange sounds at the end of the packets sent to jsmpeg?

closing and opening the audio again infinitely, turn up the volume to hear..

Any fix?

**Comment by mohamadpk** [comment]
hi luizj. the sound has  many noise. i debug it to find what is problem .1. first i think this is from push system ffmpeg.i try test it.record data on sending and seen after time.there is no problem.so 2. record data on server before and after push to client.oooh again everything is ok. 3.i record data on client when i see online my webcam.in online many noise i heard .but when i listen to recorded file in client all problem disappear. the problem is on mp2.js file and when is try to decode mp2 . i try to fix it 2 hour.so when i don't anything about coding or decoding. export is  nothing ;))

**Comment by mohamadpk** [comment]
hello... please fix sound problem

**Comment by rex-xu** [comment]
i fix this by add -re to ffmpeg like this: ffmpeg -re -i xxxxxx

**Comment by luizj** [comment]
With -re the problem is the same, the "tick" sound is almost imperceptible

2018-02-07 7:26 GMT-02:00 rex-xu <notifications@github.com>:

**Comment by mohamadpk** [comment]
yes @luizj  with -re this problem is not fix any more. the -re just re encode video and audio . As I explained earlier audio and video healthy to the destination arrive .and player play flawless "video file". but this problem is on player or decoder when that try to attach buffered section .not other pieces of program.

**Comment by sops21** [comment]
Where you able to find a fix. I'm also having this issue. If I record to a file or have the websocket-relay.js save a copy, the audio is fine when played back, but when listening over the web socket there is a ticking  noise.

**Comment by cemkoker** [comment]
Did you manage to solve the ticking noise problem ? I see this too
